src/content/en/updates/2017/01/payment-request-updates.md

```diff
@@ -0,0 +1,139 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: In Chrome 56 and 57, there are a few changes made to the Payment Request API following the spec changes. Learn what they are and make changes to your own implementations.
+
+{# wf_updated_on: 2017-01-31 #}
+{# wf_published_on: 2017-01-31 #}
+{# wf_tags: paymentrequest,chrome56,chrome57 #}
```

Comment:
We've used `payment` in the past, not `paymentrequest`. If you want to use `paymentrequest` please add it to `src/data/commonTags.json`